zip-0227.rst

```diff
@@ -184,106 +184,122 @@ Specification: Global Issuance State
 
 Issuance requires the following additions to the global state defined at block boundaries:
 
-- :math:`\mathsf{previously\_finalized}`, a set of :math:`\mathsf{AssetId}` that have been finalized (i.e.: the ``finalize`` flag has been set to ``1`` in some issuance transaction preceding the block boundary).
+- ``previously_finalized``, a set of :math:`\mathsf{AssetId}` that have been finalized (i.e.: the ``finalize`` flag has been set to ``1`` in some issuance transaction preceding the block boundary).
```

Comment:
Let's call it `finalized_assets`

Reply:
a set of `AssetBase` not `AssetId`, right?
We want a small state.

Reply:
We need finalization to work across pools. If we store `AssetBase` in this set, then the same `AssetId` in a future pool would not be finalized by default. We also cannot reverse-engineer `AssetId` from the `AssetBase` without storing the mapping...

An alternative is to use `AssetDigest` instead, as a middle ground for a smaller state?

Reply:
Good point, let's go with `AssetDigest` instead